fedimint-api/src/encoding/tbs.rs

```diff
@@ -35,6 +40,37 @@ impl_external_encode_bls!(tbs::BlindedSignatureShare, tbs::MessagePoint, 48);
 impl_external_encode_bls!(tbs::BlindedSignature, tbs::MessagePoint, 48);
 impl_external_encode_bls!(tbs::Signature, tbs::MessagePoint, 48);
 
+impl Encodable for threshold_crypto::PublicKeySet {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        let mut len = 0;
+        for coefficient in self.coefficients() {
+            len += coefficient
+                .to_affine()
+                .to_compressed()
+                .consensus_encode(writer)?;
+        }
+        Ok(len)
+    }
+}
+
+impl Encodable for threshold_crypto::PublicKey {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        self.to_bytes().consensus_encode(writer)
+    }
+}
+
+impl Encodable for tbs::AggregatePublicKey {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        self.0.to_compressed().consensus_encode(writer)
+    }
+}
+
+impl Encodable for tbs::PublicKeyShare {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        self.0.to_compressed().consensus_encode(writer)
+    }
+}
```

Comment:
:heart: 

Could we have `Decodeable` for these as well? We might get rid of `SerdeEncodable` in some places if we did.

Reply:
I'd save that for another PR, but yeah would be a good sanity check as well to see if they can be decoded.